plugins/color_panel/macos/FLEColorPanelPlugin.mm

```diff
@@ -47,9 +47,13 @@ - (void)handleMethodCall:(FLEMethodCall *)call result:(FLEMethodResult)result {
 /**
  * Configures the shared instance of NSColorPanel and makes it the frontmost & key window.
  */
-- (void)showColorPanel {
+- (void)showColorPanel:(id)args {
```

Comment:
Why is this id? It's strongly discouraged to have APIs that don't have meaningful type information, and I don't see any reason it should be necessary here.

Also, absent any expectation for lots more arguments (as we have in the open/save panel), I'd prefer the argument be extracted before this point, and the API here take an opacity bool argument rather than a dictionary of parameters.

Reply:
Yeah I thought there might be a of parameters eventually, but looking closer to the api, they are not that many. Passing bool directly should work.